backend/backend/utils/eeljokes.py

```diff
@@ -4,10 +4,10 @@
 db = SQLAlchemy()
 
 class EelJokes(db.Model):
-    __tablename__ = 'EelJokes'
-    JokeID = db.Column(db.Integer, primary_key=True)
-    JokeText = db.Column(db.Text, nullable=False)
-    JokeTextLine2 = db.Column(db.Text, nullable=True)
+    __tablename__ = 'jokes'
+    id = db.Column(db.Integer, primary_key=True, default=db.text("(SELECT MAX(id) + 1 FROM jokes)"))
```

Comment:
You could try to use the `__tablename__` variable instead of "jokes" so if the table name ever changes, you only have to change it in the one spot.

Reply:
IMO this isn't a valuable enough refactoring. I would prefer the queries be as straightforward as possible re: variable substitution  only for things that should actually be parameterized etc...